package.json

```
@@ -9,12 +9,12 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon -e ts --exec 'ts-node src/index.ts'",
+    "start": "nodemon -e ts --exec ts-node src/index.ts",
     "debug": "ts-node --inspect --compilerOptions '{\"inlineSources\":true}' src/index.ts",
-    "build:editor": "cd src/editor && yarn && npm run build && cd -",
+    "build:editor": "cd src/editor && yarn && npm run build && cd .. && cd ..",
```

```suggestion
    "build:editor": "cd src/editor && yarn && npm run build && cd ../..",
```
Can you please test it on Windows?

---

Indeed it does work, novadays windows does not care, thank you for suggestion